src/streamlink/plugins/ard_live.py

```diff
@@ -1,56 +1,68 @@
 import re
+import json
```

Comment:
there is also `parse_json`

https://github.com/streamlink/streamlink/blob/1b436da7942e492cfbb40f5088deec4ce5d4a3f9/src/streamlink/utils/__init__.py#L48